bcos-sync/bcos-sync/interfaces/BlockSyncMsgFactory.h

```diff
@@ -41,7 +42,7 @@ class BlockSyncMsgFactory
     virtual BlockSyncStatusInterface::Ptr createBlockSyncStatusMsg(
         bcos::protocol::BlockNumber _number, bcos::crypto::HashType const& _hash,
         bcos::crypto::HashType const& _gensisHash, int64_t const time = utcTime(),
-        int32_t _version = 0)
+        int32_t _version = static_cast<int32_t>(BlockSyncMsgVersion::v1))
```

Comment:
Not set the default version to `v1` here, pass the version when calling `createBlockSyncStatusMsg` instead.